android/capacitor/src/main/java/com/getcapacitor/PluginsConfig.java

```diff
@@ -0,0 +1,118 @@
+package com.getcapacitor;
+
+import org.json.JSONObject;
+
+/**
+ * Represents the configuration options for plugins used by Capacitor
+ */
+public class PluginsConfig {
```

Comment:
I think we may have misunderstood each other. I was envisioning a reusable `PluginConfig` class that would correspond to a _single_ plugin's configuration within the Capacitor config file. 

Usage would look like this:

```java
config.getPluginConfig(TEST_PLUGIN_NAME).getString("var2")
```